Api1/src/main/java/com/project2/api1/Services/OrdersService.java

```diff
@@ -104,12 +107,17 @@ public int createOrder(OrderDTO o)
         {
             lst.add(menuItemRepository.findById(i).get());
         }
-        Orders order = new Orders(99,o.getCustomerId(), LocalDateTime.now(),null,"New",false,lst);
+        Orders order = new Orders(99,o.getCustomerId(), LocalDateTime.now(),null,"New",lst);
         return ordersRepository.save(order).getOrderId();
     }
 
-    public boolean placeOrder(int id)
+    public String placeOrder(int oId)
     {
-    return true;
+        Customer c = customerRepository.findById(ordersRepository.findById(oId).get().getCustomerId()).get();
+        RestTemplate tmp = new RestTemplate();
+        HttpEntity<TransferOrderDTO> req = new HttpEntity<>(new TransferOrderDTO(oId,c.getContactType(),c.getId(),c.getEmailAddress(),c.getPhoneNumber()));
+        String resourceUrl = "http://localhost:8000/resteraunt/getOrder";
```

Comment:
Sorry, restaurant is spelled incorrectly in the endpoint. 
The correct endpoint would be: 
`http://localhost:8000/restaurant/getOrder`